scripts/bootstrap-celery.sh

```diff
@@ -72,11 +85,13 @@ cd eventkit-cloud
 sudo git fetch origin
 cp -R * /var/lib/eventkit
 cd /var/lib/eventkit
-sudo apt-get -y install libxml2-dev libxslt-dev
-export CPLUS_INCLUDE_PATH=/usr/include/gdal
-export C_INCLUDE_PATH=/usr/include/gdal
-pip install -r requirements.txt
-pip install -r requirements-dev.txt
+## EDIT DOCKER_COMPOSE IF NEEDED
+sudo docker-compose up -d celery
+#sudo apt-get -y install libxml2-dev libxslt-dev
+#export CPLUS_INCLUDE_PATH=/usr/include/gdal
```

Comment:
also curious about code being commented out v/ deleted outright?  what is the criterion you're applying for this decision?

/me is curious so I have a rule of thumb in my travels within the codebase